apps/backend/src/datasources/mockups/FileDataSource.ts

```diff
@@ -44,4 +46,8 @@ export default class FileDataSourceMock implements FileDataSource {
       new Date()
     );
   }
+
+  async getBlobdata(fileName: string): Promise<ReadStream | null> {
+    return null;
+  }
```

Comment:
Are we mocking only returning null or  if filename if provided we must return information.

Reply:
This will be the default case when the data for that particular proposal id is not present in db right. In that case, it will return null. If needed, I can change this to return a stream.

Reply:
It must potentially return a ReadStream from its signature.